templates/web/pages/bmviii_home_temp.tmpl

```diff
@@ -0,0 +1,265 @@
+                        <img alt="Lauren Lum" src={{ static_path "lauren.jpg" }}>
+                    </div>
+                    <div class="column is-half" style="word-wrap: break-word">
+                        <p>
+                            BoilerMake gave me the resources and support I needed to create a project I was truly proud of.
+                            If you have an idea and a passion for creating something from scratch, don’t let this opportunity pass you up!
+                        </p>
+                        <p style="font-family: Cabin; family-style: normal; font-weight: 600; font-size: 20px" >
+                            Lauren Lum
+                        <p>
+                    </div>
+                </div>
+                <div class="container carousel-entry has-background-image" style='display: flex; justify-content: space-between; background-image: url({{ static_path "bmviii_testimonial1_background.svg" }})'>
+                    <div class="column is-half">
+                        <img alt="Samuel Ingram" src={{ static_path "sam.jpg" }}>
+                    </div>
+                    <div class="column is-half" style="word-wrap: break-word">
+                        <p>
+                            BoilerMake gave me the resources and support I needed to create a project I was truly proud of.
+                            If you have an idea and a passion for creating something from scratch, don’t let this opportunity pass you up!
+                        </p>
+                        <p style="font-family: Cabin; family-style: normal; font-weight: 600; font-size: 20px" >
+                            Samuel Ingram
+                        <p>
+                    </div>
+                </div>
+                <div class="container carousel-entry has-background-image" style='display: flex; background-image: url({{ static_path "bmviii_testimonial1_background.svg" }});  justify-content: space-between'>
+                    <div class="column is-half">
+                        <img alt="Kevin Cardona" src={{ static_path "kevin.jpg" }}>
+                    </div>
+                    <div class="column is-half" style="word-wrap: break-word">
+                        <p>
+                            BoilerMake gave me the resources and support I needed to create a project I was truly proud of.
+                            If you have an idea and a passion for creating something from scratch, don’t let this opportunity pass you up!
+                        </p>
+                        <p style="font-family: Cabin; family-style: normal; font-weight: 600; font-size: 20px" >
+                            Kevin Cardona
+                        <p>
+                    </div>
+                </div>
+                <div class="carousel-button-next"></div>
+                <div class="carousel-button-prev"></div>
+            </div>
+        </div>
+    </div>
+</section>
```

Comment:
Transition between testimonials and sponsors is harsh, seems like sponsors section has the wrong background color?
![Screen Shot 2020-10-05 at 21 25 06](https://user-images.githubusercontent.com/10781907/95148459-f25aaa80-0748-11eb-8220-b2b076701247.png)